smartsim/experiment.py

```diff
@@ -38,7 +38,7 @@
 from .generation import Generator
 from .settings import settings
 from .utils import get_logger
-from .utils.helpers import colorize, init_default
+from .utils.helpers import colorize, detect_launcher, init_default
```

Comment:
Is a user ever going to want to use this function? If so, we may considering creating a new module thats more user facing as the helpers are going to be moved to the ``_core`` directory.

I think it's possible a user may want to inspect the value returned by ``detect_launcher`` for specific use cases. I could go either way on this one. If you don't think so, just leave it.

If you do think we should have a module, names?

``expUtils``?
``wlmUtils``?
``wlm``? 

I could see the last one being good if we end up moving the slurm (allocation) api to the ``wlm`` module.